### PR TITLE
Add support for displaying PR labels

### PR DESCRIPTION
**Pull request by collisdigital**
Fetches each PR as a GitHub issue (required to get the labels array) and displays any labels found on the wall in their GitHub chosen colour.

Example screenshot:

screen shot 2016-12-08 at 00 14 41

### CODE REVIEW

File: javascript/core.js [hunk 1]
```diff
@@ -146,4 +146,6 @@
   );
   FourthWall.importantUsers = [];
 
+  FourthWall.showLables = FourthWall.getQueryVariable('labels') === 'true';
```
Comment (ajmaddaford) [inline javascript/core.js]: Typo in showLables

**Review by ajmaddaford — CHANGES_REQUESTED** [review]

File: javascript/core.js (hunk 1 above)
Comment (collisdigital) [reply javascript/core.js]: Done

**Review by ajmaddaford — APPROVED** [review]

**Review by jonnyshaw89 — APPROVED** [review]

**Comment by robyoung** [comment]
Any plans to submit this upstream? Looks really useful.

**Comment by collisdigital** [comment]
We're doing quite a few changes to fourth-wall locally at the moment and have broken (or not updated) the unit tests etc. When we finish the batch of changes and fix the tests we'll raise a PR. You can use this fork for now?

**Comment by collisdigital** [comment]
See also https://github.com/ONSdigital/fourth-wall/pull/4

**Comment by collisdigital** [comment]
Alternatively if you feel like fixing the Jasmine tests here and raising a PR we'd gladly accept :-D

**Pull request merged**